grpc-client/client.go

```diff
@@ -87,45 +99,89 @@ func main() {
 			CaCertFile: *stopCaCertFile,
 		}
 
-		stopResponse, err := Stop(models.StopRequest{TaskID: *stopTaskID},
-			*stopHost, permission)
+		if (*stopAlias != "") == (*stopTaskID != "") {
+			fmt.Println("error: must provide one (and only one) of task_id or alias")
+			os.Exit(1)
+		}
+
+		var stopResponse *models.StopResponse
+		if *stopAlias != "" {
+			stopResponse, err = Stop(models.StopRequest{Alias: *stopAlias},
```

Comment:
You don't need an extra attribute for the alias - the taskID could host both - this way you only differentiate in the server.